### PR TITLE
update the shell doc, add deploy + init instructions, windows bits, etc

### PR DESCRIPTION
**Pull request by mikedotexe**
I mainly wanted to add about the deploy + initialize feature that I just shipped, but made a few changes along the way.
Discussed how some commands and environment variables work on Windows, etc.

### CODE REVIEW

File: docs/roles/developer/contracts/shell.md [hunk 1]
```diff
@@ -153,50 +174,94 @@ If the account was used to authorize an example app like Greeting or Counter ava
...
+```bash
+near deploy --wasmFile out/contract.wasm --accountId my-contract.YOU.testnet
 ```
 
-**Finally test** one of the contract methods
+## Deploy and initialize a contract
 
-```text
-near call name-of-contract-account._???_.testnet nameOfContractMethod --account-id _???_.testnet
+Many contracts (especially those written in Rust) will have an initialization method that is typically run immediately after deploying. This method is oftentimes called `new` and takes a couple arguments.
```
Comment (thisisjoshford) [inline docs/roles/developer/contracts/shell.md:210]: minor grammar edit... 

will have an initialization method that ~~is~~ typically run***s*** immediately after deploying.

File: docs/roles/developer/contracts/shell.md [hunk 2]
```diff
@@ -153,50 +174,94 @@ If the account was used to authorize an example app like Greeting or Counter ava
...
+near deploy --accountId my-contract.YOU.testnet --wasmFile res/fungible_token.wasm --initFunction new --initArgs '{"owner_id": "my-contract.YOU.testnet", "total_supply": "100000000000000000"}'
+```
+
+**Note**: in Windows, single quotes have issues and the above command would be:
+
+```bash
+near deploy --accountId my-contract.YOU.testnet --wasmFile res/fungible_token.wasm --initFunction new --initArgs "{\"owner_id\": \"my-contract.YOU.testnet\", \"total_supply\": \"100000000000000000\"}"
+```
+
+Lastly, there are two extra commands for deploy + initialize, `initGas` and `initDeposit` that can be used. Be sure to use the `--help` flag to see more information about these and other flags/options. For deploy-specific help, you may enter:
```
Comment (thisisjoshford) [inline docs/roles/developer/contracts/shell.md:235]: another minor edit...

```suggestion
Lastly, there are two extra commands for deploy + initialize (`initGas` and `initDeposit`) that can be used. Be sure to use the `--help` flag to see more information about these and other flags/options. For deploy-specific help, you may enter:
```

**Comment by thisisjoshford** [comment]
Made some minor grammar suggestions (see above) and also noticed something outside of your PR on line 278 needs adjustment (see below)

`view` methods don't change the state of the blockchain, as opposed to `change` methods which do, and therefore require a cryptographic signature(.) **This** cost***s*** a little more to compute and take**s** a little longer to process since ~~they're~~ **their** execution is recorded permanently on the blockchain (which requires consensus)**.**

File: docs/roles/developer/contracts/shell.md (hunk 1 above)
Comment (mikedotexe) [reply docs/roles/developer/contracts/shell.md:210]: thanks, adding this to PR with another internal suggestion

**Pull request merged**